### PR TITLE
fix: pytorch dataloader yields tensor batches

### PR DESCRIPTION
**Pull request by bolgheroni**

### CODE REVIEW

File: src/spira_training/shared/adapters/model_trainer/pytorch_model_trainer/pytorch_audio.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import torch
 
 PytorchWav = NewType("PytorchWav", torch.Tensor)
+PytorchLabel = NewType("PytorchLabel", torch.Tensor)
```
Comment (lucasqml) [inline src/spira_training/shared/adapters/model_trainer/pytorch_model_trainer/pytorch_audio.py:6]: A label realmente precisa ser um torch.Tensor?

Comment (bolgheroni) [reply src/spira_training/shared/adapters/model_trainer/pytorch_model_trainer/pytorch_audio.py:6]: sim...

Comment (bolgheroni) [reply src/spira_training/shared/adapters/model_trainer/pytorch_model_trainer/pytorch_audio.py:6]: são ints na entrada, mas entendo que o pytorch transforma em tensores depois
